doc/release/README.chplenv

```diff
@@ -341,17 +341,20 @@ Optional Settings
 
 *  Optionally, the CHPL_HWLOC environment variable can select between
    no hwloc support or using the hwloc package distributed with Chapel
-   in third-party.  Current options are:
+   in third-party.  Note that hwloc is only used by the qthreads
+   tasking layer, and does not need to be built for other tasking
+   layers.  Current options are:
 
        none   : do not build hwloc support into the Chapel runtime
        hwloc  : use the hwloc distribution bundled with Chapel in
                 third-party
 
    If unset, CHPL_HWLOC defaults to "hwloc" if CHPL_TASKS is qthreads,
-   unless the target platform is knc.  In all other cases it defaults to
-   "none".  Note that the bundled hwloc distribution will not build on
-   some targets (Mac, for example), but the CHPL_HWLOC default does not
-   reflect that.
+   unless the target platform is knc.  In all other cases it defaults
+   to "none".  If the bundled hwloc distribution does not build
+   successfully, it should still be possible to use qthreads so long
+   as CHPL_LOCALE_MODEL is flat.  Manually set CHPL_HWLOC to "none" and
```

Comment:
Ah, I didn't realize this. We actually get link errors if we use Qthreads with LOCALE_MODEL=numa and libnuma cannot be found? I thought the Qthreads config step was in control of whether or not it built itself to use libnuma.

(Note that the old hardcoded -lnuma the runtime/etc/Makefile.hwloc-hwloc used to throw doesn't matter if CHPL_HWLOC=none. Plus that line in Makefile.hwloc-hwloc isn['t there any more since I moved the decision
making about what libraries to link in into the util/chplenv scripts.)

For the performance issue, it would probably be enough to say "the performance will be awful" (politely).  :-)